libs/blocks/table/table.css

```diff
@@ -322,14 +322,16 @@
 }
 
 /* icons */
-.table:not(.merch) .row .col span.milo-tooltip {
-  margin-left: var(--spacing-xs);
-  margin-right: 0;
+[dir='ltr'] .table:not(.merch) .row .col span.milo-tooltip {
```

Comment:
I don't think we need `[dir='ltr']` here, we can just write as:
```cs
.table:not(.merch) .row .col span.milo-tooltip {
  margin-right: -4px;
}
```
and make sure that on `[dir='rtl']` we reset the `margin-right` value to `0`
```css
[dir='rtl'] .table:not(.merch) .row .col span.milo-tooltip {
  margin-right: 0;
  margin-left: -4px;
}
```